release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -65,6 +65,8 @@ Because {product-title} {product-version} now uses a {op-system-base} 9.2 based
 
 * Some component configuration options and services might have changed between {op-system-base} 8.6 and {op-system-base} 9.2, which means existing machine configuration files might no longer be valid.
 
+* Notably, if you had customized the default openssh server configuration (`/etc/ssh/sshd_config`), you will have to update it according to link:https://access.redhat.com/solutions/7030537[KB7030537].
```

Comment:
```suggestion
* If you customized the default OpenSSH `/etc/ssh/sshd_config` server configuration file, you must update it according to this link:https://access.redhat.com/solutions/7030537[Red Hat Knowledgebase article].
```
(please do not commit changes with this UI tool, it messes up the ability to squash commits)